Source/Core/DolphinQt2/Config/Mapping/MappingWidget.cpp

```diff
@@ -31,6 +31,28 @@ MappingWindow* MappingWidget::GetParent() const
   return m_parent;
 }
 
+bool MappingWidget::IsIterativeInput() const
+{
+  return m_parent->IsIterativeInput();
+}
+
+void MappingWidget::NextButton(MappingButton* button)
+{
+  auto iterator = std::find(m_buttons.begin(), m_buttons.end(), button);
+
+  iterator++;
+
+  if (iterator == m_buttons.end())
+    return;
+
+  auto* next = *iterator;
```

Comment:
Since it's not obvious what `next` is in this case, please write the type explicitly: `MappingButton*`